src/Compositor.hpp

```diff
@@ -39,39 +39,41 @@ class CCompositor {
     std::string                                m_szCurrentSplash     = "error";
 
     std::vector<PHLMONITOR>                    m_vMonitors;
-    std::vector<PHLMONITOR>                    m_vRealMonitors; // for all monitors, even those turned off
+    std::vector<PHLMONITOR>                    m_vRealMonitors; // for all monitors.
```

Comment:
the "even those turned off" comment led me quite astray for a while. But its not accurate, so I removed it.

I'm actually not sure there's any difference between these two vectors at this point.

Reply:
how is it not accurate? If you have a monitor that is `disable`d in the config, it will be in real but not in monitors.

Reply:
turned off monitors seem to get removed from that list. At least, with my specific hardware they do.

Reply:
that might be a weird controller on your display, but in general, they shouldnt